tests/foreman/cli/test_role.py

```diff
@@ -426,81 +431,832 @@ def test_positive_update_role_taxonomies(self):
+
+        :steps:
+
+            1. Create Org Admin role and assign Org A and Location A
+            2. Create new user A and assign Org A and Location A
+            3. Assign Org Admin role to User A
+            4. Create another user B and assign Org A and Location A
+            5. Assign any role to user B that does have access to Org A and
+                Location A
+            6. Login with Org Admin user A and attempt to view user B
+
+        :expectedresults: Org Admin should be able to access users inside
+            its taxonomies
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier2
+    def test_negative_access_users_outside_org_admin_taxonomies(self):
+        """Org Admin can not access users outside its taxonomies
+
+        :id: 81081c75-031d-4aca-acd6-25868a492a84
+
+        :steps:
+
+            1. Create Org Admin role and assign Org A and Location A
+            2. Create new user A and assign Org A and Location A
+            3. Assign Org Admin role to User A
+            4. Create another user B and assign Org B and Location B
+            5. Assign any role to user B that doesnt have access to Org A and
+                Location A
+            6. Login with Org Admin user A and attempt to view user B
+
+        :expectedresults: Org Admin should not be able to access users outside
+            its taxonomies
+
+        :CaseLevel: Integration
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_create_taxonomies_by_org_admin(self):
+        """Org Admin cannot define/create organizations and locations
+
+        :id: 115c46ea-f2fc-4be0-bbdb-26faf9246809
+
+        :steps:
+
+            1. Create Org Admin role and assign any taxonomies to it
+            2. Create user and assign above Org Admin role to it
+            3. Login with above Org Admin user
+            4. Attempt to create Organizations and Locations
+
+        :expectedresults: Org Admin should not have access to create taxonomies
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_access_all_global_entities_by_org_admin(self):
+        """Org Admin can access all global entities in any taxonomies
+        regardless of its own assigned taxonomies
+
+        :id: 6ebccf86-1766-432a-ad7c-4f2f606e1604
+
+        :steps:
+
+            1. Create Org Admin role and assign Org A and Location A
+            2. Create new user and assign Org A,B and Location A,B
+            3. Assign Org Admin role to User
+            4. Login with Org Admin user
+            5. Attempt to create all the global entities in org B and Loc B
+                e.g Architectures, Operating System
+
+        :expectedresults: Org Admin should have access to all the global
+            entities in any taxonomies
+        """
+
+    @stubbed
+    @tier3
+    def test_positive_access_entities_from_ldap_org_admin(self):
+        """LDAP User can access resources within its taxonomies if assigned
```

Comment:
LDAP validation can have another concept than usual users. Did you check CLI outputs to know whether taxonomies placed in same tables and can be easily fetched.

Anyway we can add setup section here that AD should be configured in order to work with these test cases